packages/client/vite.config.ts

```diff
@@ -68,16 +73,7 @@ export default defineConfig(({ mode }) => {
   }
 
   return {
-    /*
-     * https://github.com/storybookjs/storybook/issues/18920
-     * the issue occurs because of util.js which is a
-     * transitive depedency of storybook. I think it might
-     * be a good idea to separate components and storybook
-     * in that case because possibly storybook is getting
-     * included in components bundle
-     */
```

Comment:
@Zangetsu101 I think this is not needed anymore as it's related to Webpack according to [this comment](https://github.com/storybookjs/storybook/issues/18920#issuecomment-1310273755)

Reply:
Seems to be working fine in the end-to-end environment.